pyface/fields/toggle_field.py

```diff
@@ -0,0 +1,22 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+#
+# Author: Enthought, Inc.
+# Description: <Enthought pyface package component>
```

Comment:
nit: do we really need these or were they copy-pasted from some other module?

Reply:
Copy-pasted.  I don't think we've done the header normalization for pyface: some of the copyright dates were also out of sync.